worker-pools.yml

```diff
@@ -228,6 +228,34 @@ pools:
             - *scratch-disk
             - *scratch-disk
           machine_type: c2-standard-16
+  - pool_id: '{pool-group}/b-linux-docker-alpha'
+    description: Worker for Firefox automation.
+    owner: release+tc-workers@mozilla.com
+    email_on_error: true
+    variants:
+      - pool-group: gecko-1
+    provider_id: fxci-level1-gcp
+    config:
+      minCapacity: 0
+      maxCapacity: 50
+      regions: [us-central1, us-west1]
+      image: ubuntu-2404-headless-alpha
+      implementation: generic-worker/linux-d2g
+      worker-config:
+        genericWorker:
+          config:
```

Comment:
Do you want to disable native payloads on this pool?

Reply:
Being an `-alpha` pool, I was thinking not, to allow more messing around.

Reply:
fair!